source/Calamari.Azure/Calamari.Azure.csproj

```diff
@@ -0,0 +1,106 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>1.0.0.0</Version>
+    <TargetFramework>net452</TargetFramework>
```

Comment:
Before the conversion this project targeted net45, why does it need to be upgraded now?